README.md

```diff
@@ -313,14 +313,14 @@ questions that helped us define what this project is all about. This is part of
 these questions: If you have a question that you think could shape this
 project please PR this document or open an issue!
 
-### Why wouldn't you just use passport?
+### Why wouldn't you just use a passport?
```

Comment:
you can revert this one, passport is the name of a js library :)

Reply:
Ok